### PR TITLE
Update the ImageClassification and KNN load and save

### PR DESCRIPTION
**Pull request by cvalenzuela**
Follows [ml5-library#99](https://github.com/ml5js/ml5-library/pull/99):

- Updating the ImageClassifier method in the examples.
- Updating the library reference to 0.0.3
- Adding save and load buttons to the KNNImageClassifier

### CODE REVIEW

File: p5js/00_ImageClassification/index.html [hunk 1]
```diff
@@ -6,7 +6,8 @@
 
   <script src="https://cdnjs.cloudflare.com/ajax/libs/p5.js/0.6.0/p5.min.js"></script>
   <script src="https://cdnjs.cloudflare.com/ajax/libs/p5.js/0.6.0/addons/p5.dom.min.js"></script>
-  <script src="https://unpkg.com/ml5@0.0.2/dist/ml5.js"></script>
+  <script src="https://unpkg.com/ml5@0.0.3/dist/ml5.js"></script>
```
Comment (shiffman) [inline p5js/00_ImageClassification/index.html]: Love this workflow!

File: p5js/02_KNNImageClassification/sketch.js [hunk 2]
```diff
@@ -16,6 +16,18 @@ function setup() {
 }
 
 function createButtons() {
+  // Save and Load buttons
+  save = select('#save');
+  save.mousePressed(() => {
```
Comment (shiffman) [inline p5js/02_KNNImageClassification/sketch.js]: Should probably remove `=>` (for now at least)

**Comment by shiffman** [comment]
So when these examples are changed, is the correct workflow to then update the [website repo](https://github.com/ml5js/ml5js.github.io) as well? (I'm happy to work on that just checking.)

**Comment by cvalenzuela** [comment]
Yep, I just pushed changes to reflect this on the website

File: p5js/02_KNNImageClassification/sketch.js (hunk 2 above)
Comment (cvalenzuela) [reply p5js/02_KNNImageClassification/sketch.js]: Oh, I missed that. Updated in the latest commit

**Pull request merged**